### ISSUE TITLE
Is it possible to pass groups with transactions not to be signed?

### ISSUE THREAD

**Issue by ivanschuetz** [randlabs/myalgo-connect#50]
WalletConnect allows to pass group transactions that are not to be signed by the current user (i.e. only for informative purpose). See e.g [tutorial](https://developer.algorand.org/docs/get-details/walletconnect/):

```js
const txnsToSign = txns.map(txn => {
  const encodedTxn = Buffer.from(algosdk.encodeUnsignedTransaction(txn.txn)).toString("base64");

  return {
    txn: encodedTxn,
    message: 'Description of transaction being signed',
    // Note: if the transaction does not need to be signed (because it's part of an atomic group
    // that will be signed by another party), specify an empty singers array like so:
    // signers: [],
  };
});
```

This way it's possible for the user to review all the transactions in the group when signing their part.

Is this possible with My Algo?

**Comment by Alex99y** [comment]
Hello @ivanschuetz ! No, it isn't possible. MyAlgo Connect only receives the transactions that the user could sign.

**Comment by ivanschuetz** [comment]
Okay, I'd like to request this feature. At least for some groups it seems critical, e.g. for an atomic swap, it would make sense to be able to review the tokens you receive before you sign the "send" transaction.

**Comment by Alex99y** [comment]
@ivanschuetz  yes, we are aware of that, but it is really difficult to make this feature with the current implementation. Our long-term plan is to implement ARC-0001 which will easily allow this feature.
https://github.com/algorandfoundation/ARCs/blob/main/ARCs/arc-0001.md

**Comment by gidonkatten** [comment]
+1 for this feature

**Comment by ismael-elatifi** [comment]
Yes this feature is very much wanted !
AlgoSigner already support this :  
```js
let signedTxs = await AlgoSigner.signTxn([
  {
    txn: base64Txs[0],
  },
  {
    // This tells AlgoSigner that this transaction is not meant to be signed
    txn: base64Txs[1],
    signers: [],
  },
]);
```
https://github.com/PureStake/algosigner/blob/develop/docs/dApp-integration.md#atomic-transactions

**Comment by webbgeorge** [comment]
Hi there! Are there any current plans for this feature to be implemented? @Alex99y does MyAlgo have a public roadmap you can share?

**Comment by nkbt** [comment]
@webbgeorge 

> Hi there! Are there any current plans for this feature to be implemented? @Alex99y does MyAlgo have a public roadmap you can share?

You can just pass only unsigned txns from the group, no need to send pre-signed ones. MyAlgo is ok and does not strictly check for group validity (like AlgoSIgner does)
